plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -304,13 +304,21 @@ export class NunjucksWorkflowRunner implements WorkflowRunner {
           return;
         }
       }
+
+      const resolvedEach =
+        step.each && this.render(step.each, context, renderTemplate);
+
+      if (step.each && !resolvedEach) {
+        throw new InputError(
+          `Invalid each value passed to action ${action.id}, "${step.each}" cannot be resolved to a value`,
```

Comment:
Maybe to rephrase the error somehow? 
Or to highlight that each is a term in a data structure of action?

Reply:
Thanks @acierto, Updated the message, let me know wdyt